integration-tests/hardhat.config.ts

```diff
@@ -16,7 +16,7 @@ const config: HardhatUserConfig = {
   mocha: {
     timeout: 50000,
   },
-  solidity: '0.7.6',
+  solidity: '0.8.7',
```

Comment:
Also should use the same settings as the contracts package.

Reply:
The `contracts` package is on 0.8.7 and `integration-tests` here are on 0.8.7. What do you mean exactly?

Reply:
Answered above.